intermittent_tracker/dashboard.py

```diff
@@ -33,14 +34,13 @@ def post_attempts(request):
     return query(request)
 
 def query(request):
-    issues_db = IssuesDB.readonly()
     result = {'known': [], 'unknown': []}
     attempts = request.json['attempts']
     for attempt in attempts:
         test = {'path': attempt['path'], 'subtest': attempt.get('subtest')}
-        issues = issues_db.query(attempt['path'])
+        issues = issues_mixin(attempt['path'])
         if issues:
```

Comment:
I think one small change is needed here to ensure that the results are sorted properly between the `known` and `unknown` keys:

```suggestion
        if issues["issues"]::
```

Reply:
resolved in 930643449832a843226c18df9095aba055ad2fc4